x-pack/test/saved_object_api_integration/common/lib/saved_object_test_cases.ts

```diff
@@ -8,33 +8,60 @@ export const SAVED_OBJECT_TEST_CASES = Object.freeze({
   SINGLE_NAMESPACE_DEFAULT_SPACE: Object.freeze({
     type: 'isolatedtype',
     id: 'defaultspace-isolatedtype-id',
+    namespaces: ['default'],
```

Comment:
I'm a bit torn on adding the `namespaces` field to the common test cases. They are only used for the `find` suite. Perhaps these attributes should be moved to the `find` suite instead?

Reply:
Yeah I went back and forth here too. My reasoning for including them here was to keep the test cases consolidated. If we moved these to the `find` suite, then we would either have to duplicate these cases just to add `namespaces`, or we'd have to do a large conditional block to add the right `namespaces` value based on the saved object id in the test case.

There might be another option I'm overlooking though...did you have something else in mind?

Reply:
Something like this in `x-pack/test/saved_object_api_integration/common/suites/find.ts`:

```diff
const TEST_CASES = [
  { ...CASES.SINGLE_NAMESPACE_DEFAULT_SPACE, namespaces: ['default'] },
  { ...CASES.SINGLE_NAMESPACE_SPACE_1, namespaces: ['space_1'] },
  { ...CASES.SINGLE_NAMESPACE_SPACE_2, namespaces: ['space_2'] },
  { ...CASES.MULTI_NAMESPACE_DEFAULT_AND_SPACE_1, namespaces: ['default', 'space_1'] },
  { ...CASES.MULTI_NAMESPACE_ONLY_SPACE_1, namespaces: ['space_1'] },
  { ...CASES.MULTI_NAMESPACE_ONLY_SPACE_2, namespaces: ['space_2'] },
  { ...CASES.NAMESPACE_AGNOSTIC, namespaces: undefined },
  { ...CASES.HIDDEN, namespaces: undefined },
];

...

-const allCases = Object.values(CASES);
+const allCases = TEST_CASES;
```